scripts/depcheck.js

```diff
@@ -32,7 +32,7 @@ const EXEC_OPTIONS = { silent: true, async: true }
         result.package = pkg
         result.packagePath = packagePath
         return result
-    }))).filter((result) => Object.keys(result.missing).length)
+    }))).filter((result) => Object.keys(result.missing).length && Object.keys(result.missing) != 'puppeteer')
```

Comment:
@christian-bromann Added a filter. Do you have any better ways to filter it?

Reply:
Let's define a constant that defines all deficits we accept and match them against `package` and `packagePath`.